packages/dashboard-frontend/.storybook/main.js

```diff
@@ -0,0 +1,56 @@
+import { dirname, join } from "path";
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that use Yarn PnP or are set up within a monorepo.
+ * @param {string} value The package name or path.
+ * @returns {string} The absolute path to the package.
+ */
+function getAbsolutePath( value ) {
+	return dirname( require.resolve( join( value, "package.json" ) ) );
+}
+
+/** @type { import("@storybook/react-webpack5").StorybookConfig } */
+const config = {
+	stories: [
+		"../stories/**/*.mdx",
+		"../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)",
+	],
+	addons: [
+		getAbsolutePath( "@storybook/addon-webpack5-compiler-swc" ),
+		getAbsolutePath( "@storybook/addon-essentials" ),
+		getAbsolutePath( "@storybook/addon-interactions" ),
```

Comment:
See also the comment for `package.json`:
This is for the play functionality which is not used in the stories at this point, do we want to keep it?